### PR TITLE
feature/swipe to complete checkin in guestlist

### PR DESCRIPTION
**Pull request by mhlangagc**
### Closes Issues:
- #81 
- #82 

### Release video link
https://cl.ly/580f69bf45ed

### Pods Installed
- pod 'SwipeCellKit', '2.5.4'

### CODE REVIEW

File: Big Neon Core/Services/DatabaseService/DatabaseService.Scan.swift [hunk 1]
```diff
@@ -152,8 +152,8 @@ extension DatabaseService {
     
     public func fetchUpdatedGuests(forEventID eventID: String, changeSince: String?, completion: @escaping (_ error: Error?, _ updatedGuestsDict: [[String: Any]]?) -> Void) {
         
-        
         let apiURL = APIService.fetchEvents(eventID: eventID, changesSince: changeSince, page: nil, limit: nil, query: nil)
+        print(apiURL)
```
Comment (Krakaw) [inline Big Neon Core/Services/DatabaseService/DatabaseService.Scan.swift]: Do we need this print here?

File: Big Neon Core/Utilities/DateConfig.swift [hunk 2]
```diff
@@ -55,9 +55,15 @@ final public class DateConfig {
...
+    public class func stringFromDate(date: Date) -> String {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
+        return dateFormatter.string(from: date)
+    }
+    
     public class func serverDateFromDate(stringDate: Date) -> String? {
         let dateFormatter = DateFormatter()
-        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss"
+        dateFormatter.dateFormat = "yyyy-MM-dd'UTC'HH:mm:ss"
```
Comment (Krakaw) [inline Big Neon Core/Utilities/DateConfig.swift]: Does the formatter require the timezone in place of the T as that is a string literal?

File: Big Neon/Screens/Explore/GuestListViewController/GuestsListViewModel.swift [hunk 3]
```diff
@@ -77,21 +68,23 @@ final class GuestsListViewModel {
...
         if changeSince == nil {
-            date = DateConfig.serverDateFromDate(stringDate: Date())!
-//            date = DateConfig.formatServerDate(date: date, timeZone: eventTimeZone)
+            date = DateConfig.stringFromDate(date: Date())
         } else {
             date = changeSince!
         }
         
-        BusinessService.shared.database.fetchUpdatedGuests(forEventID: eventID, changeSince: date) { [weak self] (error, guestsFetched) in
+        print(date)
```
Comment (Krakaw) [inline Big Neon/Screens/Explore/GuestListViewController/GuestsListViewModel.swift]: Print required?

File: Big Neon/Screens/Explore/GuestListViewController/GuestsListViewModel.swift [hunk 4]
```diff
@@ -77,21 +68,23 @@ final class GuestsListViewModel {
...
+        
+        BusinessService.shared.database.fetchUpdatedGuests(forEventID: eventID, changeSince: date) { (error, guestsFetched) in
             DispatchQueue.main.async {
                 guard var guests = guestsFetched else {
                     completion(false)
                     return
                 }
                 
+                print(guests)
```
Comment (Krakaw) [inline Big Neon/Screens/Explore/GuestListViewController/GuestsListViewModel.swift]: Print?

File: Big Neon/Screens/Explore/TicketScannerViewController/ScannerViewController.AutoScan.swift [hunk 5]
```diff
@@ -6,12 +6,18 @@ import AVFoundation
...
 extension ScannerViewController {
     
-    func checkinAutomatically(withTicketID ticketID: String, fromGuestTableView: Bool, atIndexPath: IndexPath?) {
+    func checkinAutomatically(withTicketID ticketID: String, fromGuestTableView: Bool, atIndexPath indexPath: IndexPath?) {
         
         self.scannerViewModel?.automaticallyCheckin(ticketID: ticketID) { [weak self] (scanFeedback, errorString, ticket) in
             DispatchQueue.main.async {
                 if fromGuestTableView == true {
-                    self?.guestListVC?.reloadGuests(atIndex: atIndexPath!)
+                    if self?.guestListVC?.isSearching == true {
```
Comment (Krakaw) [inline Big Neon/Screens/Explore/TicketScannerViewController/ScannerViewController.AutoScan.swift]: Should we set local variables for the `self?.guestListVC?` to keep it DRY?

**Comment by Krakaw** [comment]
Minor comments, otherwise looks good.

**Comment by mhlangagc** [comment]
@Krakaw I've updated the swipe to checkin feature and removed the bugs you picked up. If you possibly can, please review the app on a device (I can show it to you on mine). The swipe to checkin feature doesn't swipe the whole way (though it could) but there is haptic feedback to let the user know they have swiped in the user.

**Pull request merged**